wpcc.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
 
   s.test_files = Dir['spec/**/*']
 
-  s.add_dependency 'dough-ruby'
+  s.add_dependency 'dough-ruby', '~> 5.25'
```

Comment:
I was trying to decide where to write up the documentation for this. There's a gotcha, that was happening on Frontend, but in relevance to this. I can add it to the README here on wpcc? 
This PR should fix the gotcha though. 

Updating the version I can put some documentation on the README here.